persistance/remind.ts

```diff
@@ -0,0 +1,139 @@
+import {
+    IPersistence,
```

Comment:
Again this code is not Not relevant to this app

Reply:
deleted